File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Python/PRAK605-2310817120009-AlysaArmelia.py
matriks_n = int(input())

matriks_kolom1 = []
matriks_kolom2 = []
matriks_kolom3 = []

print ("Matriks A")

for baris in range(matriks_n):
    deret = list(map(int, input().split()))
    matriks_kolom1.append (deret)

print("Matriks B")

for baris in range(matriks_n):
    deret = list(map(int, input().split()))
    matriks_kolom2.append (deret)

for baris in range (matriks_n):
    matriks_kolom3.append ([0] * matriks_n)

for baris1 in range (matriks_n):
    for baris2 in range (matriks_n):
        for baris3 in range (matriks_n):
            matriks_kolom3 [baris1][baris2] += matriks_kolom1 [baris1][baris3] * matriks_kolom2 [baris3][baris2]

print("Matriks AXB")

for baris1 in range (matriks_n):
    for baris2 in range (matriks_n):

        print(matriks_kolom3 [baris1][baris2], end=" ")
    
    print()

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Bahasa C/PRAK605-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main()
{
    int matriks_n;

    scanf("%d", &matriks_n);

    int matriks_kolom1 [matriks_n][matriks_n];
    int matriks_kolom2 [matriks_n][matriks_n];
    int matriks_kolom3 [matriks_n][matriks_n];

    printf("Matriks A\n");

    int baris1;
    int baris2;
    int baris3;

    for (baris1 = 0; baris1 < matriks_n; ++baris1)
    {
        for (baris2 = 0; baris2 < matriks_n; ++baris2)
        {
            scanf("%d", &matriks_kolom1[baris1][baris2]);
        }
    }
    printf("Matriks B\n");

     for (baris1 = 0; baris1 < matriks_n; ++baris1)
    {
        for (baris2 = 0; baris2 < matriks_n; ++baris2)
        {
            scanf("%d", &matriks_kolom2[baris1][baris2]);
        }
    }

    int sum = 0;

    for (baris1 = 0; baris1 < matriks_n; baris1++)
    {
        for (baris2 = 0; baris2 < matriks_n; ++baris2)
        {
            for (baris3 = 0; baris3 < matriks_n; ++baris3)
            {
                sum+= matriks_kolom1[baris1][baris3] * matriks_kolom2[baris3][baris2];
            }
            matriks_kolom3[baris1][baris2] = sum;
            sum = 0;
        }

    }

    printf("Matriks AXB\n");

    for (baris1 = 0; baris1 < matriks_n; ++baris1)
    {
        for (baris2 = 0; baris2 < matriks_n; ++baris2)
        {
            printf("%d ", matriks_kolom3[baris1][baris2]);
        }
        printf("\n");
    }
return 0;
}